dockerfiles/worker.Dockerfile

```diff
@@ -21,6 +21,9 @@ RUN apt-get update && \
       pkg-config \
       software-properties-common \
       unzip \
+      libsm6 \
```

Comment:
Which Python libraries depend on these Linux dependencies? I believe we should dynamically & conditionally install these dependencies instead of adding them to the worker Docker image like that, since most models probably won't require this. The place to edit should be https://github.com/nginyc/rafiki/blob/dev/rafiki/model/utils.py#L55.

Reply:
Cv2 Python library needs these 3 dependencies. You give a good suggestion.